test/libraries/cusparse.jl

```diff
@@ -52,6 +52,7 @@ blockdim = 5
     @test size(d_x,3) == 1
     @test ndims(d_x)  == 2
     CUDA.@allowscalar begin
+        @test sprint(show, MIME"text/plain"(), d_x) == replace(sprint(show, MIME"text/plain"(), x), "SparseMatrixCSC{Float64, Int64}"=>"CuSparseMatrixCSC{Float64, Int32}")
```

Comment:
Bit specific, but I guess this works for now (until we implement our own show method).